wheatley/types.py

```diff
@@ -0,0 +1,9 @@
+""" A module to hold common type aliases """
+
+from typing import Any, Dict, List
+from wheatley.bell import Bell
+
+Call = Dict[int, str]
```

Comment:
Did you consider / try using 
```python
Call = NewType('Call', Dict[int, str])
```
which makes it a distinct sub type (https://docs.python.org/3/library/typing.html#newtype)

I can't tell reviewing online if this would be more helpful (stop assigning something incorrectly) or a hinderence (prevent assigning a looser typed JSON or literal with a cast or something)

Reply:
I think for everything other than `JSON` this would make more sense.  I hadn't really looked at how `NewType`s worked, but I'll do that now.

Reply:
Also, should this be the plural `Calls`?

Reply:
That's a point. Though it's actually closer to the _definition_ of a call, so perhaps `CallDef`?

Reply:
Or perhaps `CallType`?

Reply:
Ah yes, this represents a single word, e.g. 'Bob', normally with one effect, but sometimes multiple at different points.
`CallDef` is probably the better one

Reply:
`CallDef` was my favourite too, so `CallDef` it is.

Reply:
Done!